spearhead/assets/sass/_accesibility.scss

```diff
@@ -0,0 +1,38 @@
+/* Text meant only for screen readers. */
+.screen-reader-text {
```

Comment:
Rather than copy/paste this from Seedlet, can we import it as a helper, like we do for the navigation CSS?

Reply:
oh you are totally right, I was unsure if this was the correct way to do it. I will fix!

Reply:
As we discussed, I turned it into a mixin within seedlet and imported it on spearhead